docs/Cli.md

```diff
@@ -338,6 +338,7 @@ After restoring it's always a good idea to `dump` or `diff` the settings once ag
 |  fw_iterm_throw_limit  | 165 | Limits max/min I-term value in stabilization PID controller in case of Fixed Wing. It solves the problem of servo saturation before take-off/throwing the airplane into the air. By default, error accumulated in I-term can not exceed 1/3 of servo throw (around 165us). Set 0 to disable completely. |
 |  fw_reference_airspeed  | 1000 | Reference airspeed. Set this to airspeed at which PIDs were tuned. Usually should be set to cruise airspeed. Also used for coordinated turn calculation if airspeed sensor is not present. |
 |  fw_turn_assist_yaw_gain  | 1 | Gain required to keep the yaw rate consistent with the turn rate for a coordinated turn (in TURN_ASSIST mode). Value significantly different from 1.0 indicates a problem with the airspeed calibration (if present) or value of `fw_reference_airspeed` parameter |
+|  fw_loiter_direction  | RIGHT | Direction of loitering: center point on right wing (clockwise - default), or center point on left wing (counterclockwise) |
```

Comment:
IMO it is confusing to specify clockwise/counter-clockwise which is relative to the point of view. Center on right/left wing is enough and always the same no matter the point of view.

Reply:
I found it important to specify "clockwise / counter-clockwise" due to the author of the issue and to make it clear that the "clockwise" is on the right and the counter-clockwise on the left.